app/main.py

```diff
@@ -1,3 +1,81 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: None) -> "Distance":
```

Comment:
The type annotation for the `other` parameter should not be `None`. It should be `Union[Distance, int, float]` to indicate that it can accept a `Distance` object or a numeric type.